src/scss/components/_succeed_page.scss

```diff
@@ -0,0 +1,42 @@
+.succeed {
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  background-color: $primary-color;
+  width: 100%;
+  min-height: 40vh;
+  box-sizing: content-box;
+
+  &__text {
+    box-sizing: border-box;
+    width: 50%;
+    padding-left: 10%;
+    color: #ffffff;
+    z-index: 1;
+
+    &__p {
+      margin: 2vh;
+    }
+
+    &__huge {
```

Comment:
theo BEM thì, 
- tên class của Block và Element phải là danh từ
- tên class của Modifier là tính từ, kiểu --primary, --active, --is-highlight, ...